features/project/views/new_project_form.erb

```diff
@@ -4,27 +4,78 @@
       <span class="mdl-layout__title"><%= title %></span>
     </div>
   </header>
-
+  <script>
+  document.addEventListener("DOMContentLoaded", function() {
```

Comment:
Why do we all of this on the front-end?

Reply:
So the main idea was to first display the form of `New Project` as we have today, but now provide a list of `branches` to select. Whenever the user selects a new `branch`, we request to the backend the `lanes` and `platforms` existing for a given repo and a given branch.